dts/bindings/iio/adc/nxp,vf610-adc.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2021, Antonio Tessarolo
+# SPDX-License-Identifier: Apache-2.0
+
+description: Vf610 Adc
+
+compatible: "nxp,vf610-adc"
+
+include: adc-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    "#io-channel-cells":
+      const: 1
+
+    rdc:
+      type: int
+      required: true
+      description: Set the RDC permission for this peripheral
```

Comment:
Some more explanation of what to put in here would be good if this is user-settable.